src/css/_card.scss

```diff
@@ -0,0 +1,87 @@
+.deck {
+  display: grid;
+  grid-gap: 1rem; // sass-lint:disable-line no-misspelled-properties
+  grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+  padding: 1rem;
```

Comment:
I believe these magic numbers should be moved to variables in another file and imported. This for example should become `std-padding` or similar and reused.

Reply:
not a fan of std-padding as a concept as padding and margin can be quite content specific